package.json

```diff
@@ -47,6 +48,8 @@
     "utils"
   ],
   "dependencies": {
+    "@nuxt/types": "^2.14.7",
+    "@types/socket.io-client": "^1.4.34",
```

Comment:
I think these should go in "devDependencies"?

Reply:
Oh wait, I think dependencies is fine.

Reply:
Yes, these need to be in dependencies I think, so they get downloaded automatically along with `nuxt-socket-io`